docs/linode-writers-formatting-guide/index.md

```diff
@@ -11,6 +11,7 @@ modified_by:
 published: 2014-01-15
 title: Linode Writer's Formatting Guide
 show_on_rss_feed: false
+type: "guide"
```

Comment:
this is the line that makes it searchable in Algolia. Basically, if this line is not set, then Hugo by default will set the [content type](https://gohugo.io/content-management/types/) to "page". The code in our theme (for some historical reason that I can't determine) skips all pages with type `page` when building the index.json that gets shipped to Algolia